### PR TITLE
Fix issue #13389

### LINKED ISSUE #13389: Typo in 'revolver ammo casing' for Generic Guns mod
Revolver ammo itself is correct, but the casing is called 'revovler ammo casing'.

http://puu.sh/jLMO7/b414a8f577.png

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26103143-typo-in-revolver-ammo-casing-for-generic-guns-mod?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Griffinhart**
Corrects a typo ("revovler" -> "revolver"), fixes #13389.

### CODE REVIEW

File: data/mods/Generic_Guns/gg_ammo_override.json [hunk 1]
```diff
@@ -426,4 +426,4 @@
         "count" : 10,
         "effects" : ["INCENDIARY"]
     }
-]
```
Comment (Griffinhart) [inline data/mods/Generic_Guns/gg_ammo_override.json]: No idea why it says that I touched this line when I didn't.

Comment (i2amroy) [inline data/mods/Generic_Guns/gg_ammo_override.json]: What file did you open this in and edit it? What it's saying is the that the ending newline character on the last line was removed, which some text editors will do automatically.

**Comment by chaosvolt** [comment]
Hmm. Mildly annoying to have to go to the issue page to see what this PR does. :V

**Comment by Griffinhart** [comment]
Updated the PR comment.

File: data/mods/Generic_Guns/gg_ammo_override.json (hunk 1 above)
Comment (Griffinhart) [inline data/mods/Generic_Guns/gg_ammo_override.json]: vim.

Comment (BevapDin) [inline data/mods/Generic_Guns/gg_ammo_override.json]: vim (:+1:) adds the newline, it isn't in the current master branch.

A newline at the end of a text file is standard in the Unix world. See for example http://unix.stackexchange.com/questions/18743.

Basically: Unix considers a line to be *terminated* by a newline, which is thereby part of the line. Windows considers the newline to be a line *separator*, the last line does not need to be separated from anything and therefor does not need a newline here.

Editors use/enforce/encourage whatever they like, many can be configured for one or the other behavior.

> e: cross-checking with the original file, I don't see a trailing newline (in the original, I mean - both my PR and the master are 429 lines).

Yes, some Unix tools may consider the last line to be technically "incomplete" without the newline character. The program you used for line counting does apparently not care about this. But other line counting tools may simply count the number of newline characters and therefor give a different result.

Comment (Griffinhart) [inline data/mods/Generic_Guns/gg_ammo_override.json]: > The program you used for line counting does apparently not care about this.
Vim for my copy, going to https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/mods/Generic_Guns/gg_ammo_override.json in my browsers (Firefox and Iron) for the master.

**Pull request merged**